gui/src/main/java/de/azubi/kniffel/gui/fx/controllerhandling/GameController.java

```diff
@@ -0,0 +1,500 @@
+            scoreLabel.setText("Du hattest schon 3 mal gewürfelt!");
+
+        }
+    }
+
+    //alle würfel werfen, welche nicht "behalten" werden
+    private void moveToRolling() {
+        moveDiceToRolling(dice1, 0);
+        moveDiceToRolling(dice2, 1);
+        moveDiceToRolling(dice3, 2);
+        moveDiceToRolling(dice4, 3);
+        moveDiceToRolling(dice5, 4);
+    }
+
+    private void moveDiceToRolling(ImageView dice1, int index) {
+        if (this.gfx.rou.dices[index].keep() && keptDice.getChildren().contains(dice1)) {
+            keptDice.getChildren().remove(dice1);
+            rollingDice.getChildren().add(dice1);
+        }
+    }
+
+    // ergebniss anzeigen, letztes ergebniss löschen
+    private void updateScore(Score[] scoreboards, int currentPlayer) {
+        tableView.getItems().clear();
+        int[][] scores = new int[MAX_PLAYERS][];
+        for (int player = 0; player < MAX_PLAYERS; player++) {
+            scores[player] = scoreboards[player].getScore();
+        }
+        int[] possibleScore = scoreboards[0].getMaxScore();
+
+        for (int i = 0; i < MAX_SCORE_NAME; i++) {
+            String[] scoreText = new String[MAX_PLAYERS];
+
+            for (int player = 0; player < MAX_PLAYERS; player++) {
+                String textPossibleScore = possibleScore[i] == 0 ? "" : Integer.toString(possibleScore[i]);
+                String textRealScore = scores[player][i] == -1 ? "" : Integer.toString(scores[player][i]);
+                if (currentPlayer == player && "".equals(textRealScore) && !"".equals(textPossibleScore)) {
+                    scoreText[player] = textPossibleScore + "R";
+                } else {
+                    scoreText[player] = textRealScore;
+                }
+            }
+            tableView.getItems().add(new ScoreRow(ScoreUtils.lower(i + 1), scoreText));
+        }
+
+    }
+
+    // ergbniss anzeigen ohne klickbare fläche
+    private void updateScoreRealOnly(Score[] scoreboards) {
+        tableView.getItems().clear();
+
+
+        // ergebnisliste auch bei mehreren Spielern anzeigen
+
+//TODO Namen selber bestimmen können
+        int[][] scores = new int[MAX_PLAYERS][];
+        for (int player1 = 0; player1 < MAX_PLAYERS; player1++) {
+            scores[player1] = scoreboards[player1].getScore();
+        }
+
+        for (int player2 = 0; player2 < MAX_PLAYERS; player2++) {
+            scores[player2] = scoreboards[player2].getScore();
+        }
+
+        for (int player3 = 0; player3 < MAX_PLAYERS; player3++) {
+            scores[player3] = scoreboards[player3].getScore();
+        }
+
+        for (int player4 = 0; player4 < MAX_PLAYERS; player4++) {
+            scores[player4] = scoreboards[player4].getScore();
+        }
+
+        for (int player5 = 0; player5 < MAX_PLAYERS; player5++) {
+            scores[player5] = scoreboards[player5].getScore();
+        }
+
+
+        for (int i = 0; i < MAX_SCORE_NAME; i++) {
```

Comment:
Wenn Player1 bis player-n in einem Array leben würden könnte man hier auch eine Schleife nutzen.